tests/common.bash

```diff
@@ -6,7 +6,10 @@ load "../helpers/bats-support/load.bash"
 load "../helpers/bats-assert/load.bash"
 
 function kubectl() {
-	command kubectl --context $CLUSTER_CONTEXT "$@"
+	local smartargs
+	[[ "$@" =~ ^(apply|delete) ]] && smartargs="--wait=true "
+	[[ "$@" =~ ^(apply|delete|wait|rollout\ status) ]] && smartargs+="--timeout=5m"
+	command kubectl ${smartargs:-} --context=$CLUSTER_CONTEXT "$@"
```

Comment:
While I like the idea, I'm worried that by overloading `kubectl` with more things besides `--context`, things are not gonna be obvious in the future.

I have an habit for taking lines out of the bats and running them outside of bats when iterating on something (maybe because running the whole bats file takes too long). And at first glance I wouldn't realize that the `--wait` is gone.

Reply:
I do not know what I think about the smart args. But I would say it is okay.

Reply:
Sorry for late answer, I am already changing it - just takes longer to figure out where we actually need to wait and what's happening during that. 
`kubectl apply/delete` --wait is not helping much, I will remove it.
I am figuring out about `--timeout=5m`.

Reply:
I removed smartargs and added `wait_for` function, can you check again please?